src/jellyfin.ts

```diff
@@ -24,7 +24,9 @@ export function startJellyfinUnmonitor(app: Express) {
       switch (Item.Type) {
         case 'Episode': {
           const episodeTvdbIds = [Item.ProviderIds.Tvdb];
-          const seriesTitle = Series.OriginalTitle;
+          const titleFoundById = await getSeriesTitleById(req.body);
+
+          const seriesTitle = titleFoundById ?? Series.OriginalTitle;
 
           await unmonitorEpisode({ episodeTvdbIds, seriesTitle });
```

Comment:
Instead of adding these functions to get the series title, just modify unmonitorEpisode to take in seriesProviderIds  so it can do a series match on that as a first pass.

The only reason unmonitorEpisode works off of the series title is because this was originally built for Plex and its webhook doesn't expose any series metadata.